chore(site): add team page

with-heart opened the pull request:
This PR adds a team page to the site, fetching the team's data using `gatsby-source-graphql` and the GitHub API.

![image](https://user-images.githubusercontent.com/1954752/83950641-518bcb00-a7fa-11ea-80c0-f8efb50fb8bc.png)

In order for this to work, we need to make sure we have a `GITHUB_API_TOKEN` in env with `repo` and `read:org` permissions available in the build step.

segunadebayo commented on docs/src/pages/team.js:
```diff
@@ -0,0 +1,104 @@
+}
+
+const sortMembers = (a, b) => {
+  // segun comes first!
```
💖

segunadebayo commented:
Thanks for this PR @with-heart.

Can you also include the list of all contributors below that? I think Github has a way of showing all the code contributors to a repo.

If it's possible, let's do this and possibly filter out the core members since we're already at the top :)

What do you think?

with-heart commented:
@segunadebayo Love it! I'll implement that shortly.

with-heart commented:
According to [this post on github.community](https://github.community/t/graphql-get-repository-contributors/14422/7), there is no way to get the contributors. I can just read .all-contributorsrc though, so I'll go that route :)

with-heart commented:
Added a list of contributors. Each contributor displays an avatar that links to the user's github profile page. 

Updated screenshot:

![image](https://user-images.githubusercontent.com/1954752/83971338-ea791f80-a8a8-11ea-8363-168ca32a121c.png)
